django_project/ta_system/views.py

```diff
@@ -12,13 +13,16 @@ def home(request):
     if request.method not in ('GET', 'POST'):
         return handle_bad_request(request, app='ta_system', expected='GET, POST')
 
+    context = {
+        'system_is_open': SystemStatus.objects.order_by('id').last()
```

Comment:
Same question here as for admin.py -- can we order by the 'date_changed' field instead?

Reply:
Also, why can't we just pass the status, rather than passing the whole SystemStatus instance? I remember you said something over Slack about this change fixing a bug I had run into when I pulled down your branch, but I don't really understand why

Reply:
If I'm trying to pass the status of an instance when there's no instance of SystemStatus (before admin makes any change to the SystemStatus), it will throw an error of "'NoneType' object has no attribute 'status'" on line 17 in the views.py. However, it will work properly for me to test if an attribute is true for a null object, and return False rather than an error

Reply:
Ah I see. You could still manage this in Python using a Try/Except, but I think the small decrease in the readability of the template variable is worth not having to mess with a try/except clause in the view.